### PR TITLE
BITMAKER-2079 estela-web: Implement new "SpiderPage"

### PR DESCRIPTION
**Pull request by reqhiem**
# Description

Implement "In Project/ Spiders" mockup.

# Issue

https://tasks.bitmaker.dev/issues/2079

## Checklist before requesting a review

- [ ] I have performed a self-review of my code.
- [ ] My code follows the style guidelines of this project.
- [ ] I have made corresponding changes to the [documentation](https://github.com/bitmakerla/estela/tree/main/docs).
- [ ] New and existing tests pass locally with my changes.
- [ ] If this change is a core feature, I have added thorough tests.
- [ ] If this change affects or depends on the behavior of other _estela_ repositories, I have created pull requests with the relevant changes in the affected repositories. Please, refer to our [official documentation](https://estela.bitmaker.la/docs/).
- [ ] I understand that my pull request may be closed if it becomes obvious or I did not perform all of the steps above.

### CODE REVIEW

File: estela-web/src/components/SpiderListPage/index.tsx [hunk 1]
```diff
@@ -1,12 +1,14 @@
-import React, { Component } from "react";
+import React, { Component, Fragment, ReactElement } from "react";
 import { Link } from "react-router-dom";
-import { Layout, List, Pagination, Typography } from "antd";
+import { Col, Layout, Row, List, Pagination, Typography, Button, Space, Table, Tag } from "antd";
```
Comment (raymond1242) [inline estela-web/src/components/SpiderListPage/index.tsx:3]: Remove unused imports

File: estela-web/src/components/SpiderListPage/index.tsx [hunk 2]
```diff
@@ -1,12 +1,14 @@
-import React, { Component } from "react";
+import React, { Component, Fragment, ReactElement } from "react";
```
Comment (raymond1242) [inline estela-web/src/components/SpiderListPage/index.tsx:1]: Remove unused imports

File: estela-web/src/components/SpiderListPage/index.tsx [hunk 3]
```diff
@@ -1,12 +1,14 @@
...
 import { Link } from "react-router-dom";
-import { Layout, List, Pagination, Typography } from "antd";
+import { Col, Layout, Row, List, Pagination, Typography, Button, Space, Table, Tag } from "antd";
+import { PlusOutlined } from "@ant-design/icons";
```
Comment (raymond1242) [inline estela-web/src/components/SpiderListPage/index.tsx:4]: There is an already custom Icon for this import, review assets/icon folder

**Review by raymond1242 — CHANGES_REQUESTED** [review]
Please check the comments

**Comment by reqhiem** [comment]
I solved the observations

File: estela-web/src/components/SpiderListPage/index.tsx [hunk 4]
```diff
@@ -1,12 +1,13 @@
...
 import { RouteComponentProps } from "react-router-dom";
 
 import "./styles.scss";
 import { ApiService, AuthService } from "../../services";
 import { ApiProjectsSpidersListRequest, Spider } from "../../services/api";
 import { authNotification, resourceNotAllowedNotification, Header, ProjectSidenav, Spin } from "../../shared";
+import { ColumnsType } from "antd/lib/table";
 
 const { Content } = Layout;
 const { Title } = Typography;
```
Comment (raymond1242) [inline estela-web/src/components/SpiderListPage/index.tsx:13]: This import is unused

**Review by raymond1242 — CHANGES_REQUESTED** [review]
Last observations to fix

File: estela-web/src/components/NotificationsInboxPage/styles.scss [hunk 5]
```diff
@@ -0,0 +1,14 @@
...
+
+// .create-new-project {
+//     margin: 30px auto;
+//     display: flex;
+//     align-self: auto;
+// }
+
+// .container {
+//     background-color: #FBFCFD;
+// }
```
Comment (raymond1242) [inline estela-web/src/components/NotificationsInboxPage/styles.scss:14]: Delete comments

File: estela-web/src/components/NotificationsSettingsPage/styles.scss [hunk 6]
```diff
@@ -0,0 +1,14 @@
...
+
+// .create-new-project {
+//     margin: 30px auto;
+//     display: flex;
+//     align-self: auto;
+// }
+
+// .container {
+//     background-color: #FBFCFD;
+// }
```
Comment (raymond1242) [inline estela-web/src/components/NotificationsSettingsPage/styles.scss:14]: Delete comments

File: estela-web/src/services/api/generated-api/models/Spider.ts [hunk 7]
```diff
@@ -37,6 +37,16 @@ export interface Spider {
...
+    /**
+     * Last run time.
+     * @type {string}
+     * @memberof Spider
+     */
+    lastRun?: string;
+    /**
+     * Number of jobs
+     */
+    jobs?: number;
```
Comment (raymond1242) [inline estela-web/src/services/api/generated-api/models/Spider.ts:49]: Don't add fields on files generated

**Review by raymond1242 — CHANGES_REQUESTED** [review]
Check and fix the comments

**Comment by reqhiem** [comment]
@raymond1242 it's ready to review

**Review by raymond1242 — APPROVED** [review]
Good Job! It's ready to merge

**Comment by raymond1242** [comment]
@reqhiem When you clicked merge. make sure you select `Squash and Merge` option

**Pull request merged**